apps/bare-expo/scripts/start-ios-e2e-test.ts

```diff
@@ -211,3 +214,26 @@ async function ensureDirAsync(dirPath: string) {
     }
   }
 }
+
+async function setTargetDevice() {
+  const { stdout } = await spawnAsync('xcodebuild', [
+    '-workspace',
+    './ios/BareExpo.xcworkspace',
+    '-scheme',
+    'BareExpo',
+    '-showdestinations',
+  ]);
+
+  const firstDevice = stdout.split('\n').filter((line) => line.includes('name:iPhone'))[0];
+  const deviceName = firstDevice.replace(/^.*name:(iPhone[^}]*) .*$/, '$1');
+
+  return deviceName || TARGET_DEVICE;
+}
+
+async function setTargetIOSVersion() {
```

Comment:
```suggestion
async function queryIosSdkVersionAsync() {
```

this method does not set anything